dev-packages/e2e-tests/test-applications/standard-frontend-react/tests/error-test.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { expect, test } from '@playwright/test';
+import { waitForError, waitForTransaction } from '../event-proxy-server';
+
+test('Sends an exception to Sentry after a pageload and attaches the transaction info', async ({ page }) => {
+  const errorPromise = waitForError('standard-frontend-react', async errorEvent => {
+    return !errorEvent.type;
+  });
+
+  await page.goto(`/`);
+
+  const [, error] = await Promise.all([page.locator('#exception-button').click(), errorPromise]);
+
+  expect(error).toMatchObject({
+    exception: {
+      values: [
+        {
+          type: 'Error',
+          value: 'I am an error!',
+          mechanism: {
+            type: 'generic',
+            handled: true, // called via captureException, this makes sense
+          },
+        },
+      ],
+    },
+    transaction: '/',
+    contexts: {
+      trace: {
+        data: {
+          'sentry.source': 'route',
+          'sentry.origin': 'auto.pageload.react.reactrouter_v6',
+          'sentry.op': 'pageload',
+          'sentry.sample_rate': 1,
+        },
+        op: 'pageload',
+        span_id: expect.any(String),
+        trace_id: expect.any(String),
+        origin: 'auto.pageload.react.reactrouter_v6',
+      },
+    },
+  });
+});
```

Comment:
This checks two things:

1. That the trace context is applied correctly to the error event (this is inherently tied to the active root span)
2. That the `transaction` field is set correctly (this is no longer tied to the active root span at all)